backend/src/main/java/com/huellapositiva/application/dto/ChangePasswordDto.java

```diff
@@ -0,0 +1,26 @@
+package com.huellapositiva.application.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class ChangePasswordDto {
+
+    @NotEmpty(message = "You must to write something")
+    @NotBlank(message = "The password is not blank space")
```

Comment:
If minimum size is 6 and the space character is not allowed, then it cannot be blank by definition.